notifications/src/main/java/com/dojo/notifications/model/SlackMethodsLookupByEmail.java

```diff
@@ -0,0 +1,46 @@
+package com.dojo.notifications.model;
+
+import com.hubspot.slack.client.methods.JsonStatus;
+import com.hubspot.slack.client.methods.MethodWriteMode;
+import com.hubspot.slack.client.methods.RateLimitingTier;
+import com.hubspot.slack.client.methods.RateLimitingTiers;
+import com.hubspot.slack.client.methods.SlackMethod;
+
+public enum SlackMethodsLookupByEmail implements SlackMethod {
```

Comment:
Class name SlackMethodsLookupByEmail and enum name users_lookupByEmail are misleading. 
Are there going to be any other enums declared in this class?
Enums are typed with upper cases